src/julia-syntax.scm

```diff
@@ -2021,7 +2021,7 @@
      (if (has-parameters? (cddr e))
          (error (string "unexpected semicolon in \"" (deparse e) "\"")))
      (if (any assignment? (cddr e))
-         (syntax-deprecation "assignment inside T{ }" "" #f))
+         (error (string "invalid assignment location \"" (deparse e) "\"" )))
```

Comment:
"assignment location" sounds to me like the left-hand side of the assignment is invalid. Maybe something like "misplaced assignment statement in ...".

Reply:
I just went with terminology I saw in other places, didn't really give it much thought. Will change.